### DISCUSSION TITLE
Solidity 101, Section 3 FundMe, Lesson 29, Deploy to Zksync, gives me error "underlying network change"

### DISCUSSION THREAD

**Discussion by BowtiedHarpyEagle** [Cyfrin/foundry-full-course-cu#2602] [Q&A]
Hi, 

I'm going through this course again after the Zksync and quiz updates, to make sure I'm not missing anything. 

My code is on my github: 

https://github.com/BowtiedHarpyEagle/cu-sol101-03-FundMe

The error I am getting reads:

Deploying contract FundMe
Error: "underlying network changed (event=\"changed\", network={\"chainId\":324,\"name\":\"zkSync Era\"}, detectedNetwork={\"name\":\"sepolia\",\"chainId\":11155111,\"ensAddress\":null}, code=NETWORK_ERROR, version=providers/5.7.2)"
>

I checked the solidity version for ZKsync plugin (0.8.24), the address for chainlink oracle for zksync sepolia test network, made sure my metamask is on the zksync sepolia network, IDK why it still says it detects sepolia network.

I'm adding a loom video as well:
https://www.loom.com/share/3dc8c04ba1ca4577b451a6a292d9780a?sid=e0194803-e741-4969-b5c2-004f13c9906e

What am I missing?

**Comment by EngrPips** [comment]
Hello @BowtiedHarpyEagle, Working with `zksync` isn't very smooth yet so if you are having a hard time with it just leave it and do all your deployment and testing on sepolia. We expect working with `zksync` to get smoother over time.

**Reply by BowtiedHarpyEagle** [reply]
Thanks. That's what I was thinking.

**Reply by EngrPips** [reply]
Yeah, That's the case. Keep crushing it.

**Reply by xi2And33** [reply]
you are so nice~
